utils/docker/Dockerfile.centos.7

```diff
@@ -12,13 +12,65 @@ LABEL maintainer="daos@daos.groups.io"
 # Intermittent cache-bust.  Used to reduce load on the actual CACHEBUST later.
 ARG CB0
 
+# If a local distro repository is supplied, then we should use it
+# and disable using the distro repositories.
+# The Docker image starts out with the DISTRO provided GPG keys
+# on the disk, but not installed.  These must be isntalled before
+# adding any repo that needs GPG keys but does not provide meta
+# data for installing those keys.  Then the epel-release
+# package must be installed and the GPG key it provides must be
+# installed.
+ARG REPO_URL=""
+ARG REPO_DISTRO=""
+RUN if [ -n "$REPO_DISTRO" ]; then                              \
+      rpm --import /etc/pki/rpm-gpg/* ;                         \
+      MY_REPO="${REPO_URL}${REPO_DISTRO}/";                     \
+      MY_NAME="${MY_REPO#*//}";                                 \
+      MY_NAME="${MY_NAME//\//_}";                               \
+      echo -e "[${MY_NAME}]\n\
+name=created from ${MY_REPO}\n\
+baseurl=${MY_REPO}\n\
+enabled=1\n\
+repo_gpgcheck=0\n\
+gpgcheck=1\n" >> /etc/yum.repos.d/local-centos-group.repo;      \
+      yum -y install --disablerepo=extras                       \
```

Comment:
I'm a bit confused.  Or maybe just mis-recalling, but I thought the catch-22 with epel-release was that it was in extras, signed by the CentOS key, and in EPEL signed by the EPEL key.  But because we don't yet have the EPEL key installed, we want to install from _extras_ which then gets us the EPEL key installed.

But in this `yum` command, you disable the _extras_ repo.

Reply:
Oh.  I see.  This is disabling the upstream extras.  The group repo being added has _extras_ in it.